cmd/embedded-cluster/install.go

```diff
@@ -434,6 +441,21 @@ var installCommand = &cli.Command{
 			Usage:  "Path to the airgap bundle. If set, the installation will be completed without internet access.",
 			Hidden: true,
 		},
+		&cli.StringFlag{
+			Name:   "http-proxy",
+			Usage:  "HTTP proxy to use for the installation",
+			Hidden: true,
+		},
+		&cli.StringFlag{
+			Name:   "https-proxy",
+			Usage:  "HTTPS proxy to use for the installation",
+			Hidden: true,
+		},
+		&cli.StringFlag{
+			Name:   "no-proxy",
+			Usage:  "Comma separated list of hosts to bypass the proxy for",
+			Hidden: true,
+		},
 		&cli.BoolFlag{
 			Name:   "proxy",
 			Usage:  "Use the system proxy settings for the install operation. These variables are currently only passed through to Velero and the Admin Console.",
```

Comment:
is this usage info now out of date, since it follows the same codepath as the other proxy args?

Reply:
I'm pretty sure that `--proxy` doesn't work and never worked. It isn't passed through to the operator, so it will not have any effect.
We can clean it up in a follow-up PR, as it is used afaik by some e2e tests to mock an airgap environment.

Reply:
it's used in our CI - it gets wiped away by operator, but it worked for long enough for our airgap tests to make a snapshot